src/bin/vip-import-sql.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import chalk from 'chalk';
+
+/**
+ * Internal dependencies
+ */
+import command from 'lib/cli/command';
+import { currentUserCanImportForApp, isSupportedApp } from 'lib/site-import/db-file-import';
+import { getFileMeta, uploadFile } from 'lib/client-file-uploader';
+
+/**
+ * - Include `import_in_progress` state & error out if appropriate (this likely needs to be exposed in the data graph)
+ * - Include `hasImporterS3Credentials` & error out if false (this needs to be implemented)
+ */
+const appQuery = `
+	id,
+	name,
+	organization { id, name },
+	environments{ id, appId, type, name, primaryDomain { name } }
+`;
+
+const err = message => {
+	console.log( chalk.red( message.toString().replace( /^(Error: )*/, 'Error: ' ) ) );
+	process.exit( 1 );
+};
+
+command( {
+	appContext: true,
+	appQuery,
+	requiredArgs: 1, // TODO print proper usage example
+} ).argv( process.argv, async ( arg, opts ) => {
+	const { app } = opts;
+	const { environments, organization } = app;
+	const primaryDomainName = environments[ 0 ].primaryDomain.name;
+
+	console.log( '** Welcome to the WPVIP Site SQL Importer! **\n' );
+
+	if ( ! currentUserCanImportForApp( app ) ) {
+		err( 'The currently authenticated account does not have permission to perform a SQL import.' );
+	}
+
+	if ( ! isSupportedApp( app ) ) {
+		err( 'The type of application you specified does not currently support SQL imports.' );
+	}
+
+	console.log( 'You are about to import a SQL file to site:' );
+	console.log( `ID: ${ app.id }` );
```

Comment:
We've been using this function so far to console data: https://github.com/Automattic/vip/blob/master/src/lib/cli/format.js#L11

Do you think it makes sense to use it here too to have the same logging across the CLI?